test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -91,60 +106,96 @@ public void VerifyImages_2_1()
         }
 
         private void VerifySdkImage_NewRestoreRun(
-            string appSdkImage, string sdkImageVersion, string netcoreappVersion)
+            string appSdkImage, string sdkImageVersion, string osVariant, string netcoreappVersion)
         {
-            string sdkImage = GetDotNetImage(sdkImageVersion, DotNetImageType.SDK);
-            string buildArgs = GetBuildArgs($"netcoreapp_version={netcoreappVersion}");
-            DockerHelper.Build($"Dockerfile.{DockerHelper.DockerOS.ToLower()}.test", sdkImage, appSdkImage, buildArgs);
+            // dotnet new, restore, build a new app using the sdk image
+            List<string> args = new List<string>();
+            args.Add($"netcoreapp_version={netcoreappVersion}");
```

Comment:
Can be simplified as -

```C#
            List<string> args = new List<string>
            {
                $"netcoreapp_version={netcoreappVersion}"
            };
```

Reply:
Won't fix - Is it really?  It is more lines of code.  I don't see any real benefit.